package.json

```diff
@@ -61,13 +61,13 @@
         "lerna": "^3.22.1",
         "less": "^4.1.0",
         "less-loader": "^7.2.1",
-        "mini-css-extract-plugin": "^1.3.4",
+        "mini-css-extract-plugin": "^1.3.9",
         "minimist": "^1.2.0",
         "mkdirp": "^0.5.1",
         "npm-pack-here": "^1.2.0",
         "npm-run-all": "^4.1.3",
-        "postcss": "^7.0.19",
-        "postcss-loader": "^4.3.0",
+        "postcss": "^8.4.13",
+        "postcss-loader": "^6.2.1",
```

Comment:
These tools have many dependencies themselves and cause downstream headaches for integrators using `webpack`, yet another reason to not introduce `webpack` for this build